linera-base/src/crypto/ed25519.rs

```diff
@@ -330,7 +330,7 @@ impl Ed25519Signature {
     pub fn verify_batch<'a, 'de, T, I>(value: &'a T, votes: I) -> Result<(), CryptoError>
     where
         T: BcsSignable<'de>,
-        I: IntoIterator<Item = (&'a Ed25519PublicKey, &'a Ed25519Signature)>,
```

Comment:
Inside the `ed25519` module I wouldn't use the alias.

Reply:
Yes, found that in the next PR. Will push here.

Reply:
Actually, I'm gonna do that in the follow-up PRs to avoid waiting for the CI again.